numpy/fft/_helper.py

```diff
@@ -212,6 +214,11 @@ def rfftfreq(n, d=1.0, device=None):
     f : ndarray
         Array of length ``n//2 + 1`` containing the sample frequencies.
 
+    Raises
+    ------
+    ValueError
+        If `n` is not a positive integer.
+
```

Comment:
I don't care much, but not sure it is worth documenting such clearly wrong error (i.e. nobody should have to try/except it).